client/a8c-for-agencies/sections/purchases/controller.tsx

```diff
@@ -55,6 +60,9 @@ export const licensesContext: Callback = ( context, next ) => {
 };
 
 export const billingContext: Callback = ( context, next ) => {
+	if ( isEnabled( 'a4a-bd-checkout' ) ) {
+		window.location.href = EXTERNAL_WPCOM_BILLING_HISTORY_URL;
```

Comment:
I wonder if `window.location.replace()` would be more appropriate so that the source URL doesn't stay in the browser history. That way clicking the back button doesn't cause the redirect again. Although in my testing with Chrome it seemed ok as is. Maybe it behaves differently depending on the browser.

Reply:
Updated! @travisw